proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -97,27 +107,29 @@ message Event {
   Resource resource = 4;
   // Time of the audited event. It specifies "when" it happened.
   google.protobuf.Timestamp event_time = 5;
+  // Metadata about the audited event. It specifies any additional details about the audited event.
+  EventMetadata metadata = 6;
   // Payload of the audited event. It specifies additional context on the event.
   oneof payload {
-    PayloadOrganizationCreated organization_created = 6;
-    PayloadOrganizationDeleted organization_deleted = 7;
-    PayloadOrganizationMemberAdded organization_member_added = 8;
-    PayloadOrganizationMemberRoleChanged organization_member_role_changed = 9;
-    PayloadOrganizationMemberRemoved organization_member_removed = 10;
-    PayloadRepositoryCreated repository_created = 11;
-    PayloadRepositoryDeleted repository_deleted = 12;
-    PayloadRepositoryCommitPushed repository_commit_pushed = 13;
-    PayloadRepositoryContributorAdded repository_contributor_added = 14;
-    PayloadRepositoryContributorRoleChanged repository_contributor_role_changed = 15;
-    PayloadRepositoryContributorRemoved repository_contributor_removed = 16;
-    PayloadRepositoryVisibilityChanged repository_visibility_changed = 17;
-    PayloadPluginCreated plugin_created = 18;
-    PayloadPluginDeleted plugin_deleted = 19;
-    PayloadUserCreated user_created = 20;
-    PayloadUserDeactivated user_deactivated = 21;
-    PayloadUserDeleted user_deleted = 22;
-    PayloadUserLoggedIn user_logged_in = 23;
-    PayloadUserLoggedOut user_logged_out = 24;
+    PayloadOrganizationCreated organization_created = 7;
+    PayloadOrganizationDeleted organization_deleted = 8;
+    PayloadOrganizationMemberAdded organization_member_added = 9;
+    PayloadOrganizationMemberRoleChanged organization_member_role_changed = 10;
+    PayloadOrganizationMemberRemoved organization_member_removed = 11;
+    PayloadRepositoryCreated repository_created = 12;
+    PayloadRepositoryDeleted repository_deleted = 13;
+    PayloadRepositoryCommitPushed repository_commit_pushed = 14;
+    PayloadRepositoryContributorAdded repository_contributor_added = 15;
+    PayloadRepositoryContributorRoleChanged repository_contributor_role_changed = 16;
+    PayloadRepositoryContributorRemoved repository_contributor_removed = 17;
+    PayloadRepositoryVisibilityChanged repository_visibility_changed = 18;
+    PayloadPluginCreated plugin_created = 19;
+    PayloadPluginDeleted plugin_deleted = 20;
+    PayloadUserCreated user_created = 21;
+    PayloadUserDeactivated user_deactivated = 22;
+    PayloadUserDeleted user_deleted = 23;
+    PayloadUserLoggedIn user_logged_in = 24;
+    PayloadUserLoggedOut user_logged_out = 25;
   }
```

Comment:
Field renumbering should be okay since this is `v1alpha1` and not exposed via an RPC until BSR-510 is completed (see https://github.com/bufbuild/buf/pull/1465)